lib/dash/dash_parser.js

```diff
@@ -992,6 +991,31 @@ shaka.dash.DashParser.prototype.parseAdaptationSet_ = function(context, elem) {
 
 
 /**
+ * Indicates whether and InbandEventStream element is present at the Adaption
+ * Set or Representation level.
+ *
+ * @param {!Element} elem The AdaptationSet element.
+ * @return {boolean} Whether the InbandEventStream element if present.
+ * @private
+ */
+shaka.dash.DashParser.prototype.eventStreamIsPresent_ = function(elem) {
+  var XmlUtils = shaka.util.XmlUtils;
+
+  var adaptationEventStream = XmlUtils.findChild(elem, 'InbandEventStream');
+  var representation = XmlUtils.findChild(elem, 'Representation');
```

Comment:
This only works if there is exactly one Representation.  You should probably use findChildren instead, and loop through them.